src/middleware/dataview.middleware.js

```
@@ -0,0 +1,185 @@
+  result: 'entityCount'
+})
+
+export const setTotalPages = (req, res, next) => {
+  req.totalPages = Math.ceil(req.entityCount.count / pageLength)
+  next()
+}
+
+export const setOffset = (req, res, next) => {
+  const pageNumber = Number(req.params.pageNumber)
+  
+  if (isNaN(pageNumber)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)